docs/reference/cli-reference/cli-ref-search.md

```diff
@@ -0,0 +1,65 @@
+---
+title: NuGet CLI search command
+description: Reference for the nuget.exe search command
+author: advay26
+ms.author: t-adtand
+ms.date: 08/17/2020
+ms.topic: reference
+---
+
+# search command (NuGet CLI)
+
+**Applies to:** package consumption &bullet; **Supported versions:** 5.8+
```

Comment:
We would need to make it clear that it is available with nuget.exe only

Reply:
How might we make that more clear? From my review, this is scoped to `NuGet CLI` sections & pages only.

Reply:
my bad then... I didn't realize this page was NuGet.exe only.